doc/LOGIN_SYSTEMS.md

```diff
@@ -56,6 +59,14 @@ Or, write them in config/application.yml file
 Add the app_id and app_secret in the Jenkins and containers/docker*.yml files in the production.
 They are accessed by ENV["OAUTH_GITHUB_APP_KEY"], ENV["OAUTH_GITHUB_APP_SECRET"] etc inside the (config/initializers/omniauth.rb)[https://github.com/publiclab/plots2/blob/master/config/initializers/omniauth.rb]
 
+## How to setup login on various locations?
+
+For better UI on the website, we recently created login and signup modals. The corresponding code is present here https://github.com/publiclab/plots2/blob/master/app/views/layouts/_header.html.erb#L176-L266.
+
+The `/login` and `/signup` pages still exist as some particular features require the pages.
+
+A custom Javascript class named as `requireLogin` is active and it can be inserted in anywhere for asking the user to log in first to use that particular function. For example, it can be implemented in a button with the button class as `btn btn-default requireLogin`.
```

Comment:
Please simplify this to a single line to have better readability.
All the documentations are for active functions. So active keyword is redundant.